test/basic.test.js

```diff
@@ -51,6 +51,24 @@ test('bindings contain the name and the child bindings', async ({ same }) => {
   same(instance.bindings(), { name: 'basicTest', foo: 'bar', a: 2 })
 })
 
+test('set bindings on instance', async ({ same }) => {
+  const instance = pino({ name: 'basicTest', level: 'info' })
+  instance.setBindings({ foo: 'bar' })
+  same(instance.bindings(), { name: 'basicTest', foo: 'bar' })
+})
+
+test('newly set bindings overwrite old bindings', async ({ same }) => {
+  const instance = pino({ name: 'basicTest', level: 'info', base: { foo: 'bar' } })
+  instance.setBindings({ foo: 'baz' })
```

Comment:
What happens if you create a new child after invoking this function?

Reply:
Good question -- the new child should have the same bindings as its parent at the point of the child's creation, since these are ultimately instance properties. Later changes to the parent's bindings should not affect the child. I'll add a couple more tests to illustrate.